src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.runtimeconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "runtimeOptions": {
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "1.0.0-rc3-002733"
```

Comment:
I wonder if we shouldn't begin to tie the dotnetcli and buildtools versions together in some way.